packages/@aws-cdk/aws-cloud9/lib/environment.ts

```diff
@@ -136,7 +142,6 @@ export class Ec2Environment extends cdk.Resource implements IEc2Environment {
     }
     return new Import(scope, id);
   }
-
```

Comment:
Don't remove this line, we try to keep an empty line between functions so they are easier to visually distinguish.